tests/ppx/react/expected/aliasProps.res.txt

```diff
@@ -4,11 +4,12 @@ module C0 = {
   type props<'priority, 'text> = {priority: 'priority, text?: 'text}
 
   @react.component
-  let make = ({priority: _, ?text, _}: props<'priority, 'text>) => {
-    let text = switch text {
+  let make = (props: props<'priority, 'text>) => {
```

Comment:
Using `props` instead of destructuring to avoid the conflict from overlapped argument name with default value.